webapp/src/components/expanded_view/component.tsx

```diff
@@ -62,9 +70,21 @@ interface Props {
     callStartAt: number,
     hideExpandedView: () => void,
     showScreenSourceModal: () => void,
+    selectRhsPost?: (postID: string) => void,
+    prefetchThread: (postId: string) => void,
```

Comment:
I added some basic typing to the webapp_globals `ProductApi`, but that's somewhat prototypal and likely to change in the future. These particular prop types I think are valid because we don't use the returns and the connected action creators fulfill this type "contract/expecation," although perhaps they could be `any` instead of `void`. (Aside: I'd like to see `mattermost-redux/types/actions` get generics e.g. `Thunk<TActionResult>`, `ActionResult<TData, TError>`—I've been meaning to do that but haven't gotten to it yet. This would make it much eaiser to type these flows out.)

Reply:
Thanks!